server/utils/licenceStatus.js

```diff
@@ -439,7 +439,7 @@ function getCurfewAddressState(licence, optedOut, bassReferralNeeded, curfewAddr
       return taskStates.STARTED
     }
 
-    const required = ['cautionedAgainstResident', 'addressLine1', 'addressTown', 'postCode', 'telephone']
+    const required = ['cautionedAgainstResident', 'addressLine1', 'addressTown', 'postCode']
```

Comment:
shouldn't this be conditional based on whether the offenderIsTheMainOccupier?

Also think you are missing a test to cover this?

Reply:
The phone number is dependant on the main occupier checkbox but that is done in the Curfew Address page (LIC-1080) and not in the task list page.
I wrote unit tests that check for the existence/absence of the phone number as part of LIC-1080. 
This (task list) page doesn't actually refer to or display the phone number in any way.

Reply:
new tests included